cpu/cc26xx_cc13xx/include/cc26xx_cc13xx_wdt.h

```diff
@@ -34,8 +34,10 @@ typedef struct {
     reg32_t ICR; /**< interrupt clear */
     reg32_t RIS; /**< raw interrupt status */
     reg32_t MIS; /**< masked interrupt status */
+    reg32_t __reserved1[0x100]; /**< meh */
```

Comment:
Please add a proper commit message.
I guess the memory is really just that fragmented?

Reply:
Ow, sorry, the `meh`s were originally there, I've just copied them for uniformity. Does `reserved` sound better?

Reply:
I mean this wasn't there in `cc26x0_wdt.h`.
Or do you suggest it should always have been there?

Reply:
Yes, I've just moved them over (and added my own for new reserved fields).

Reply:
I mean they are not your own, they are from the data sheet.

![0bd5dbf2-d97e-4ed7-8a69-c2e50a1ee2ae](https://user-images.githubusercontent.com/1301112/67337399-3bf5b980-f527-11e9-871e-57d832f0dd00.png)

But yea, they should have been there in the first place and you fixed that.